cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
+      "links": [
+        {
+          "description": "All JADE runs",
+          "recid": "FIXME"
```

Comment:
Shall we have an extra record with a list of JADE run numbers? If not, this link can be deleted.